### PR TITLE
fix(Dropdown): Highlights selected option in single selection

### PR DESCRIPTION
**Pull request by silviuaavram**
Highlights selected option when opening dropdown.

Based on https://github.com/paypal/downshift/pull/652 and since now it's released, this PR will be able to perform the necessary fix.

### CODE REVIEW

**Review by layershifter — COMMENTED** [review]
Downshift has [highlightedIndex](https://github.com/paypal/downshift#highlightedindex), too. Can we implement it as autocontrolled prop?

**Comment by silviuaavram** [comment]
@layershifter let's create a new issue with making those props autocontrolled. It will be a bit more complicated, as our `default` is actually `initial` in Downshift, while `default` there is the value that prop will be used at every change (selection, close/open).

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -228,6 +229,7 @@ export default class Dropdown extends AutoControlledComponent<
       focused: false,
       searchQuery: search ? '' : undefined,
       value: multiple ? [] : null,
+      defaultHighlightedIndex: !this.props.search && !this.props.multiple ? null : undefined,
```
Comment (mnajdova) [inline src/components/Dropdown/Dropdown.tsx:233]: Can we document with some comment why we use null and why undefined?

**Review by bmdalex — APPROVED** [review]

**Review by jurokapsiar — APPROVED** [review]

File: src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (silviuaavram) [reply src/components/Dropdown/Dropdown.tsx:233]: added one. is it ok?

**Review by mnajdova — APPROVED** [review]

**Pull request merged**